src/org/thoughtcrime/securesms/preferences/ChatBackgroundActivity.java

```diff
@@ -136,13 +136,15 @@ private void scaleAndSaveImage(Context context, String destinationPath) {
             Display display = ServiceUtil.getWindowManager(context).getDefaultDisplay();
             Point size = new Point();
             display.getSize(size);
+            // resize so that the larger side fits the screen accurately
+            int largerSide = (size.x > size.y ? size.x : size.y);
```

Comment:
the parenthesis can be removed

Reply:
I'd actually leave it there because it improves readability

Reply:
@cyBerta Yes, I personally like them though and would prefer to keep them.
Its clearer that way that something is done here (instead of just an assignment), but it is just a personal style preference.